gateway/review/service.go

```diff
@@ -158,12 +177,41 @@ func (s *Service) Review(ctx *storagev2.Context, reviewID string, status types.R
 		return nil, fmt.Errorf("saving review error: %v", err)
 	}
 
-	if rev.Status == types.ReviewStatusApproved || rev.Status == types.ReviewStatusRejected {
+	// Fetch the session information
+	session, err := pgsession.New().FetchOne(ctx, rev.Session)
+	if err != nil {
+		return nil, fmt.Errorf("fetch session error: %v", err)
+	}
+
+	issueInfos := jira.AddNewReviewByUserIssueTemplate{
+		ReviewerName:         rev.ReviewOwner.Name,
+		ReviewApprovedGroups: strings.Join(approvedGroups, ", "),
+		ReviewStatus:         string(rev.Status),
+	}
+
+	jira.AddReviewByUserJiraIssue(ctx.OrgID, session.JiraIssue, issueInfos)
+
+	switch rev.Status {
+	case types.ReviewStatusApproved:
 		if err := pgsession.New().UpdateStatus(ctx, rev.Session, types.SessionStatusReady); err != nil {
 			return nil, fmt.Errorf("save sesession as ready error: %v", err)
 		}
 		// release the connection if there's a client waiting
 		s.TransportService.ReviewStatusChange(rev)
+
+		// Update JIRA issue description
+		issueInfos := jira.AddReviewReadyIssueTemplate{
+			ApiURL:    ctx.ApiURL,
+			SessionID: session.ID,
+		}
+		jira.AddReviewReadyJiraIssue(ctx.OrgID, session.JiraIssue, issueInfos)
+	case types.ReviewStatusRejected:
+		// release the connection if there's a client waiting
+		s.TransportService.ReviewStatusChange(rev)
+
+		// Update JIRA issue description
+		jira.AddReviewRejectedJiraIssue(ctx.OrgID, session.JiraIssue)
```

Comment:
I recommend logging the error.
Add a _ to indicate that you won't be dealing with the error.